bundles/specmate-auth/src/com/specmate/auth/api/IAuthenticationService.java

```diff
@@ -1,10 +1,13 @@
 package com.specmate.auth.api;
 
 import com.specmate.common.SpecmateException;
+import com.specmate.usermodel.AccessRights;
 
 public interface IAuthenticationService {
 	public String authenticate(String username, String password, String projectname) throws SpecmateException;
 	public String authenticate(String username, String password) throws SpecmateException;
 	public void deauthenticate(String token) throws SpecmateException;
 	public void validateToken(String token, String path, boolean refresh) throws SpecmateException;
+	public AccessRights getALMAccessRights(String token) throws SpecmateException;

```

Comment:
I would suggest we be more generic here: Instead of PPM and ALM I would prefer source system and target system, as in the future it could be something else (jira, trello, etc.)